app/views/episode_transcripts/_form_transcript.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="prx-uploads col-12 mb-4">
+  <div class="card shadow border-0">
```

Comment:
Since Transcripts are now in a tab, we don't need the field to be in a separate card.